listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -68,27 +67,35 @@ def get_listen_count_for_user(self, user_name, need_exact=False):
             # decode is set to False as we have not encoded the value when we set it
             # in brainzutils cache as we need to call increment operation which requires
             # an integer value
-            user_key = '{}{}'.format(REDIS_INFLUX_USER_LISTEN_COUNT, user_name)
+            user_key = '{}{}'.format(self.ns + REDIS_TIMESCALE_USER_LISTEN_COUNT, user_name)
             count = cache.get(user_key, decode=False)
             if count:
                 return int(count)
 
-        try:
-            results = self.influx.query('SELECT count(*) FROM ' + get_escaped_measurement_name(user_name))
-        except (InfluxDBServerError, InfluxDBClientError) as e:
-            self.log.error("Cannot query influx: %s" % str(e), exc_info=True)
-            raise
+        if "PYTEST_CURRENT_TEST" in os.environ:
```

Comment:
Because it doesn't really make sense to add test specific behaviour. We're not testing what'll actually be run, why bother? Maybe we should just remove the test.

Reply:
Fortunately this is no longer needed (thanks timescale, not Obama!) as continuous aggregates have been improved to always give a correct result (at the expense of a bit of performance).

Reply:
Fixed